projects/angular-reactive-validation/package.json

```diff
@@ -19,9 +19,9 @@
   "license": "MIT",
   "private": false,
   "peerDependencies": {
-    "@angular/common": "^17.0.0",
-    "@angular/core": "^17.0.0",
-    "@angular/forms": "^17.0.0",
+    "@angular/common": "^18.0.0",
```

Comment:
@davidwalschots 
I've rearranged the dependency versions of  @Angular/* to ^18.0.0.
Sorry if I made your comment disappear by fixing up  the previous commit just to keep history clean.